project_event/models/task_template.py

```diff
@@ -11,6 +11,10 @@ class TaskTemplate(models.Model):
     name = fields.Char(
         string='Name',
     )
+    resource_id = fields.Many2one(
+        'resource.resource',
+        string='Resource',
+    )
```

Comment:
@rimbendhaou: At line 27, remove store=True and  at 22 temp_resp_id is not a good id name.